tests/libtest.sh

```diff
@@ -10,7 +10,18 @@ vg_exists() {
     fi
   done
   return 1
-  
+}
+
+# Tests if the logical volume lv_name exists
+lv_exists() {
+  local lv lv_name="$1"
+
+  for lv in $(lvs --noheadings -o lv_name); do
+    if [ "$lv" == "$lv_name" ]; then
```

Comment:
Do we really have to search for lv name in output. Can we just pass the expected lv to lvs and that should tell us if it exists or not?

lvs docker-vg/docker-root-lv ?